lex.c

```diff
@@ -287,7 +287,10 @@ int yylex(void)
 			} else
 				RET('*');
 		case '/':
-			RET('/');
+			if (peek() == '=') {
+				input(); yylval.i = DIVEQ; RET(ASGNOP);
+			} else
+        RET('/');
```

Comment:
The indentation on line 293 is off ...
I also suggest splitting line 291 into three separate lines.